Library/Homebrew/cmd/casks.sh

```diff
@@ -0,0 +1,29 @@
+#:  * `casks`
+#:
+#:  List all locally installable casks including short names.
+#:
+
+homebrew-casks() {
```

Comment:
A lot of this logic is shared with `homebrew-formulae`. It might be worth having a `Library/Homebrew/something.sh` which is `source`d by both and provides a function that can be called with e.g. `Casks` and `'homebrew/cask` as arguments

Reply:
All right, I reverted all unnecessary differences in `casks.sh` and extract the common logic into `items.sh`. Better name suggestions welcome.